### ISSUE TITLE
Swagger Gen. Error description

### ISSUE THREAD

**Issue by akusmin** [domaindrivendev/Swashbuckle.AspNetCore#327]
Please make Swagger gen error description more clear for investigation.
I have some invalid api methods description and swagger returns an error:
```
System.NullReferenceException: Object reference not set to an instance of an object.
 at Swashbuckle.AspNetCore.SwaggerGen.XmlCommentsOperationFilter.<>c.<ApplyPropertiesXmlToPropertyParameters>b__11_0(ApiParameterDescription p)
 at System.Linq.Enumerable.WhereListIterator`1.MoveNext()
 at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
 at Swashbuckle.AspNetCore.SwaggerGen.XmlCommentsOperationFilter.ApplyPropertiesXmlToPropertyParameters(IList`1 parameters, ApiDescription apiDescription)
 at Swashbuckle.AspNetCore.SwaggerGen.XmlCommentsOperationFilter.Apply(Operation operation, OperationFilterContext context)
 at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreateOperation(ApiDescription apiDescription, ISchemaRegistry schemaRegistry)
 at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.CreatePathItem(IEnumerable`1 apiDescriptions, ISchemaRegistry schemaRegistry)
 at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
 at Swashbuckle.AspNetCore.SwaggerGen.SwaggerGenerator.GetSwagger(String documentName, String host, String basePath, String[] schemes)
 at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
 at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
 at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
 at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()
```
It's challenging to find a reason of an issue. Place some useful data in exception.

**Comment by dopare** [comment]
This issue is already fixed  in #316. Waiting for release

**Comment by Leowuqunqun** [comment]
@domaindrivendev  hi , I try to pull the modified code Then publish to the local Nuget server ，But the lack of UI

**Comment by ruutert** [comment]
I'm waiting for the new release .. What's the new release date ?

**Comment by domaindrivendev** [comment]
This is fixed and should be available in the latest Nuget package - 1.1.0
